content/hardware/06.nicla/boards/nicla-voice/tutorials/getting-started-ml/content.md

```diff
@@ -1,160 +1,196 @@
-
-![Serial monitor showing sound recognized](assets/phrase-recognition.svg)
-
-## Conclusion
-
-This tutorial showed how to set up the Machine Learning Tools platform to create a machine learning model to be used with the Nicla Voice. It went through how to collect audio with a phone, how to train the model with the data, and then how to build it to work with the board. Lastly, the tutorial shows how to upload the model to the Nicla Voice and how to use that model.
+---
+title: 'Getting Started with the Nicla Voice'
+description: 'Learn to use the Nicla Voice to create your own Machine Learning audio models.'
+tags: 
+  - Getting started
+  - Machine Learning tools
+author: 'Benjamin Dannegård and José Bagur'
+libraries: 
+  - name: NDP
+    url: https://github.com/edgeimpulse/firmware-syntiant-tinyml
+hardware:
+  - hardware/06.nicla/boards/nicla-voice
+software:
+  - ide-v1
+  - ide-v2
+  - web-editor
+  - iot-cloud
+---
+
+## Overview
+
+The Arduino® Nicla Voice processes audio inputs using the powerful Syntiant® NDP120™ Neural Decision Processor™, which simulates human neural pathways to execute multiple artificial intelligence (AI) algorithms and automate complex tasks. The NDP120 is capable of understanding and learning the sounds in its surroundings, recognizing different events, and detecting keywords simultaneously.
+
+To make use of these keyword triggers, such as blinking the LED when the board recognizes a specific word, a machine learning (ML) model is required. With the [Machine Learning Tools](https://cloud.arduino.cc/machine-learning-tools/) powered by Edge Impulse® and integrated into the Arduino Cloud, you can quickly build, train and deploy ML models onto the Nicla Voice. This tutorial will guide you through getting started with the Nicla Voice, testing a built-in speech recognition example, and creating your ML models with the Machine Learning Tools integrated into the Arduino Cloud.
+
+![The Nicla Voice board](assets/nicla-cover-image.svg)
+
+## Goals
+
+- Test the built-in speech recognition example
+- Learn how to capture audio data to train an ML model
+- Learn how to train an ML model using audio data
+- Learn how to export an ML model to the Nicla Voice
+- Learn how to test an ML model uploaded to the Nicla Voice
+
+## Hardware and Software Requirements
+
+### Hardware Requirements
+
+- [Nicla Voice](https://store.arduino.cc/nicla-voice) (x1)
+- Micro USB cable (x1)
+
+### Software Requirements
+
+- [Arduino IDE 1.8.10+](https://www.arduino.cc/en/software), [Arduino IDE 2.0+](https://www.arduino.cc/en/software), or [Arduino Web Editor](https://create.arduino.cc/editor)
+- To create your ML model, we will use the [Arduino Cloud](https://create.arduino.cc/iot/things) (you will need to create an account if you don't have one yet)
+- [The compiled uploaders for various operating systems and the updated NDP120 processor firmware and speech recognition model files
+- ](assets/nicla_voice_uploader_and_firmwares.zip)

```

Comment:
This is broken

![image](https://user-images.githubusercontent.com/7687624/232826970-4cf06e11-85ad-4a09-89bf-94429017853b.png)

Please fix